src/components/standalone/firewall/conntrack/ConntrackRecordsTable.vue

```diff
@@ -0,0 +1,110 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import {
+  NeButton,
+  NeTable,
+  NeTableHead,
+  NeTableHeadCell,
+  NeTableBody,
+  NeTableRow,
+  NeTableCell,
+  NePaginator,
+  useItemPagination
+} from '@nethesis/vue-components'
+import type { ConntrackRecord } from './ConntrackContent.vue'
+import { ref, defineProps } from 'vue'
```

Comment:
```suggestion
import { ref } from 'vue'
```
`defineProps` shouldn't be imported, or this warning will be printed:
```
[@vue/compiler-sfc] `defineProps` is a compiler macro and no longer needs to be imported.
```